### DISCUSSION TITLE
Linux Windows communication

### DISCUSSION THREAD

**Discussion by Ever-OvO** [tud-cor/FS19_modROS#52] [Q&A]
How does ROS in Ubuntu communicate with windows?

**Comment by gavanderhoorn** [comment]
Over TCP/IP or UDP/IP?

**Comment by Ever-OvO** [comment]
Is there a specific tutorial for this part?

**Reply by gavanderhoorn** [reply]
No. It should just work if you follow the installation instructions in [tud-cor/fs_mod_ros_windows](https://github.com/tud-cor/fs_mod_ros_windows), which is described as a dependency in [the readme](https://github.com/tud-cor/FS19_modROS#overview).

**Reply by Ever-OvO** [reply]
Thanks a lot.
